### ISSUE TITLE
Treat CRITICAL als OK when acknowledged

### ISSUE THREAD

**Issue by cxcv** [nbuchwitz/icingaweb2-module-map#69]
For discussion:
If a service is critical, the whole location is marked critical. I think it would be better to filter out acknowledged services and treat them as ok. The effect would be a map with less red flags. What do you think?

**Comment by nbuchwitz** [comment]
I agree that we have to find a solution for this as it was already asked in #66 and #54. Maybe is your suggestion to handle these acknowledged services OK the easiest way for now. I'll think about it.

**Comment by nbuchwitz** [comment]
Maybe I could add something like a exclamation mark to the symbol to indicate that there is something acknowledged

**Comment by nbuchwitz** [comment]
After thinking and talks to users at OSDC I favor a config option to decide between two solutions:

* the way it is now (red/orange/whatever icon even if a service is acknowledged)
* treat handled services as OK and display an exclamation mark as symbol

![image](https://user-images.githubusercontent.com/9132055/41352755-8c042360-6f1a-11e8-975d-0e511a1de08b.png)

**Comment by deikelenboom** [comment]
We are looking for exactly this feature. Currently it is a pain that we have to explain to non-developers that the red icons on the screen are not actually problems...

Is it still being implemented?
